include/mbedtls/x509.h

```diff
@@ -324,6 +324,133 @@ int mbedtls_x509_dn_gets(char *buf, size_t size, const mbedtls_x509_name *dn);
 
 int mbedtls_x509_string_to_names(mbedtls_asn1_named_data **head, const char *name);
```

Comment:
Pre-existing: I think this one should have been removed by #8715. It was below the "Internal module functions" marker and has no doxygen doc. Sorry for noticing only now, but could you make it internal too?